manim/mobject/graph.py

```diff
@@ -338,6 +380,68 @@ def update_edge(e, u=u, v=v):
             update_edge(edge)
             edge.add_updater(update_edge)
 
+    def _tree_layout(
+        self,
+        G: nx.classes.graph.Graph,
+        root_vertex: Union[Hashable, None],
+        scale: float,
```

Comment:
`scale` is passed, but does not seem to be used. The positions should be rescaled to make sure that the tree fits inside a box measuring scale by scale.